showcase/pom.xml

```diff
@@ -54,10 +54,11 @@
         <version>0.0.1-SNAPSHOT</version>
       </dependency>
       <dependency>
-        <groupId>junit</groupId>
-        <artifactId>junit</artifactId>
-        <version>4.13.2</version>
-        <scope>test</scope>
+        <groupId>org.junit</groupId>
+        <artifactId>junit-bom</artifactId>
+        <version>5.10.2</version>
+        <type>pom</type>
+        <scope>import</scope>
```

Comment:
As this doesn't inherit from shared-deps or pom-parent, we declare the version here.